softlayer/vm/softlayer_creator.go

```diff
@@ -41,9 +41,30 @@ func NewSoftLayerCreator(softLayerClient sl.Client, agentEnvServiceFactory Agent
 	}
 }
 
+// get the time stample of action occurs to name the vm hostname
```

Comment:
We do not include comment in code. Instead we write tests (unit and integration) that validates behavior.

The reason is simple. Comment get out of sync with code and end up being misleading. The tests (since they run all the time) tend to be more representative of behavior.

Finally, most of the comments here are not needed since the code is clear.

Only include comment when the test and the code are still not enough to make things clear.

Thanks,

@maximilien